com.nodeeffect.packwallet.yaml

```diff
@@ -0,0 +1,28 @@
+id: com.nodeeffect.packwallet
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: launch.sh
+modules:
+  - name: PackWallet
+    buildsystem: simple
+    build-commands:
+      - install.sh
+      - cp -r --remove-destination /run/build/PackWallet/bin/Release/net7.0/publish/* /app/bin/
+      - install -D launch_flatpak.sh /app/bin/launch.sh
```

Comment:
@webwarrior-ws nit: let's use longer flag names (instead of -D)

Reply:
`-D` has no longer name variant